### PR TITLE
adding IFAE facility and PIC site

### PR DESCRIPTION
**Pull request by Aco7**

### CODE REVIEW

**Comment by efajardo** [comment]
@Aco7:

Can you use this:

Facility ID: 10177
Site ID: 10271
Resoruce Group ID: 532

**Comment by Aco7** [comment]
Done.

Thank you!

File: topology/IFAE/PIC/PIC-CachingInfrastructure.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        Primary:
+          ID: bcef3fbf616e1a28248f5ad1920ec067151a44dc
+          Name: Carles Acosta
+      Security Contact:
+        Primary:
+          ID: bcef3fbf616e1a28248f5ad1920ec067151a44dc
+          Name: Carles Acosta
+    Description: StashCache cache server for PIC
+    FQDN: xcachevirgo.pic.es
+    ID: <ID>
```
Comment (efajardo) [inline topology/IFAE/PIC/PIC-CachingInfrastructure.yaml]: @Aco7 I missed this resource Id can you use:

1048

Comment (Aco7) [reply topology/IFAE/PIC/PIC-CachingInfrastructure.yaml]: Done. Thanks

**Pull request merged**